### PR TITLE
fix(typings): Added "proxy_url" to EmbedVideo

### PR DESCRIPTION
**Pull request by ray-1337**

### CODE REVIEW

**Comment by HeadTriXz** [comment]
Lgtm

File: index.d.ts [hunk 1]
```diff
@@ -529,6 +529,7 @@ declare namespace Eris {
   interface EmbedVideo {
     height?: number;
     url?: string;
+    proxy_url?: string;
```
Comment (HeadTriXz) [inline index.d.ts:531]: Sort alphabetically

**Review by HeadTriXz — CHANGES_REQUESTED** [review]

**Review by HeadTriXz — APPROVED** [review]
Lgtm

**Review by DonovanDMC — APPROVED** [review]

**Review by bsian03 — APPROVED** [review]
Won't merge just yet

**Pull request merged**